harvest/api/dummy.py

```diff
@@ -241,34 +236,28 @@ def fetch_option_market_data(self, symbol: str):
             "bid": price * 0.95,
         }
 
-    # ------------- Broker methods ------------- #
-
-    def fetch_stock_positions(self) -> List[Dict[str, Any]]:
-        raise NotImplementedError("Dummy Streamer does not support broker functions.")
-
-    def fetch_option_positions(self) -> List[Dict[str, Any]]:
-        raise NotImplementedError("Dummy Streamer does not support broker functions.")
-
-    def fetch_crypto_positions(self) -> List[Dict[str, Any]]:
-        raise NotImplementedError("Dummy Streamer does not support broker functions.")
```

Comment:
Will call their parent functions so no need to be redefined here. Replaced with a code comment of what functions where not implemented so future devs know.